runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/control/ControlClientPool.java

```diff
@@ -17,16 +17,12 @@
  */
 package org.apache.beam.runners.fnexecution.control;
 
-import org.apache.beam.sdk.fn.function.ThrowingConsumer;
-import org.apache.beam.sdk.util.ThrowingSupplier;
-
 /** Control client pool that exposes a source and sink of control clients. */
```

Comment:
<!--new_thread; commit:1fd121da1417624b3b84f0300648251da64b9cb5; resolved:0-->
This terminology ("source" and "sink" in this context) is confusing. Could you clarify more about how this is used? It might also make sense to inline the "Source" and "Sink" interfaces as inner interfaces in this one (ControlClientPool.Source,Sink)